geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/BaseSetOptions.java

```diff
@@ -47,6 +47,10 @@ public boolean isXX() {
     return exists.equals(Exists.XX);
   }
 
+  public boolean isCH() {
+    return exists.equals(Exists.CH);
+  }
+
```

Comment:
I believe it's possible to set CH and one of NX or XX, so this won't work correctly, as it makes all three options mutually exclusive. I think a boolean flag for `isCH` would be better.

Reply:
Yup that's right! That didn't make sense to group the Exists options with a Change option. I've moved the CH check out of the exists options and on its own (just a bool now).